docker-compose/.env

```diff
@@ -5,8 +5,40 @@
 # for that server in them and should not be checked in.
 # See this for more info: https://github.com/bkeepers/dotenv#can-i-use-dotenv-in-production
 
+AWS_BUCKET=beyondz-dev-canvas-files
+AWS_ACCESS_KEY_ID=THIS_VALUE_SHOULD_HAVE_BEEN_PASSED_FROM_YOUR_HOST_AKA_LOCAL_MACHINE
+AWS_SECRET_ACCESS_KEY=THIS_VALUE_SHOULD_HAVE_BEEN_PASSED_FROM_YOUR_HOST_AKA_LOCAL_MACHINE
+BZ_BASE_URL=http://canvasweb
+BZ_BITLY_ACCESS_TOKEN=12345689assdfe
+BZ_DOCUSIGN_HOST=TODO
+BZ_DOCUSIGN_API_KEY=TODO
+BZ_DOCUSIGN_API_SECRET=TODO
+BZ_DOCUSIGN_RETURN_URL=TODO
+#BZ_GOOGLE_ANALYTICS_ID=''
+BZ_HELP_URL=http://some/fake/help/url
+BZ_QUALTRICS_HOST=TODO
+BZ_QUALTRICS_API_TOKEN=TODO
+BZ_QUALTRICS_LIBRARY_ID=TODO
+# All tech/product folks exempted
+BZ_UNRESTRICTED_HTML_USERS='[1, 125, 217, 703, 1968, 2147, 2844, 2855, 3053, 3064, 3104, 3122]'
 DOMAIN=canvasweb
+ENCRYPTION_KEY=facdd3a131ddd8988b14f6e4e01039c93cfa0160
```

Comment:
idk what this actually does but seems like something that shouldn't be committed?

Reply:
Bah, i meant to drop a comment on this. It was a copy/paste fo the already checked in dev only value.  The real values are different in staging/prod. It just controls local encryption of stuff like encrypted password hashes. The only security risk I can think of is if we accidentally use the staging or prod keys.